.github/workflows/documentation-checks.yaml

```
@@ -115,14 +115,3 @@ jobs:
           install-target: ${{ inputs.install-target }}
           linkcheck-target: ${{ inputs.linkcheck-target }}
           makefile: ${{ inputs.makefile }}
```

```suggestion
          makefile: ${{ inputs.makefile }}

      # - name: Accessibility Check
      #   id: pa11y-step
      #   continue-on-error: true
      #   if: success() || failure()
      #   uses: canonical/documentation-workflows/pa11y@main
      #   with:
      #     working-directory: ${{ inputs.working-directory }}
      #     install-target: ${{ inputs.install-target }}
      #     pa11y-target: ${{ inputs.pa11y-target }}
      #     makefile: ${{ inputs.makefile }}
```

Re-add commented out pa11y check.